MbDotNet.Acceptance.Tests/AcceptanceTests/CanGetImposter.cs

```diff
@@ -0,0 +1,50 @@
+﻿using FluentAssertions;
+using MbDotNet.Models.Imposters;
+
+namespace MbDotNet.Acceptance.Tests.AcceptanceTests
+{
+    internal class CanGetImposter
+    {
+        private readonly MountebankClient _client;
+        const int ImposterPort = 6000;
+        private RetrievedImposter _retrievedImposter;
+
+        public CanGetImposter(MountebankClient client)
+        {
+            _client = client;
+        }
+
+        public void Run()
+        {
+            DeleteAllImposters();
+            CreateImposter();
+
+            GetImposter();
+
+            VerifyImposterWasRetrieved();
+
+            DeleteAllImposters();
+        }
+
+        private void GetImposter()
+        {
+            _retrievedImposter = _client.GetImposter(ImposterPort);
+        }
+
+        private void DeleteAllImposters()
```

Comment:
That's fine and the way it should be, do you have a gitter room or some other way we can have a quick talk out of posting comments on github?